mbtiles/src/bin/mbtiles.rs

```diff
@@ -212,6 +220,11 @@ async fn main_int() -> anyhow::Result<()> {
             println!("MBTiles file summary for {mbt}");
             println!("{}", mbt.summary(&mut conn).await?);
         }
+        Commands::Diff {
+            file_a: FILE_A,
+            file_b: FILE_B,
+            diff_file: DIFF,
+        } => diff(FILE_A, FILE_B, DIFF).await?,
```

Comment:
you do not need to have a dedicated `diff` function. Instead, copy the above code for `Copy` and modify it, e.g.

```rust
            let opts = MbtilesCopier {
                src_file: opts.file_a,
                dst_file: opts.diff_file,
                diff_with_file: opts.file_b,
            };
            opts.run().await?;
```

Reply:
Oh I see.  
*  Will we keep `--diff-with-file` within copy command?
* Should I remove the `diff.rs`, seems I'm walking on a totally wrong way? @nyurik

Reply:
Yeah, you don't need a separate diff function -- too complicated, and it will mostly reuse the copy code anyway.  So yes, please remove diff.rs.  In the future, we can hide the `--diff-with-file`, but keep its functionality.  The `CopyArgs` has a field for that, but we can instruct Clap to ignore that field so that users won't be able to specify it from the command line.  For now, I don't think we should remove it just yet.